src/main/java/edu/ucsb/cs156/rec/repositories/UserRepository.java

```diff
@@ -17,4 +17,11 @@ public interface UserRepository extends CrudRepository<User, Long> {
    * @return Optional of User (empty if not found)
    */
   Optional<User> findByEmail(String email);
+  
+  /**
+   * This method returns a User entity with a given email.
+   * @param name email address of the user
+   * @return Optional of User (empty if not found)
+   */
+  Optional<User> findByName(String name);
```

Comment:
See feedback on PR #56.... you are not going to need this (and it wouldn't have worked anyway).

Reply:
I'm not sure I exactly understand the changes required for #56, I don't see anything new in the code review there.